pages/linux/laptop-detect.md

```diff
@@ -0,0 +1,16 @@
+# laptop-detect
+
+> Attempt to determine if the script is running on a laptop or desktop.
+> More information: <https://gitlab.com/debiants/laptop-detect>.
+
+- Exit with 0 if likely running on a laptop, or 1 otherwise:
```

Comment:
```suggestion
- Return an exit status of 0 if the current computer is likely a laptop, else returns 1:
```

Reply:
```suggestion
- Return an exit status of 0 if the current device is likely a laptop, else returns 1:
```

1. I think `device` instead of `computer` might read better? (i.e. i can execute this on my phone, and while phones are technically computers, "device" is a more general term)

Reply:
Agreed